### ISSUE TITLE
Add a MOTD

### ISSUE THREAD

**Issue by mikeschinkel** [wplib/wplib-box#134]
Add a MOTD that announces the WPLib Box using ASCII ART and:
1. Displays the Box version
2. Displays the Linux distribution and version.
3. References three ways to learn more/get help
   
   a. See README: https://github.com/wplib/wplib-box#readme
   b. See FAQ: https://github.com/wplib/wplib-box/blob/master/FAQ.md#readme
   c. Type:  `wplib help`

**Comment by mikeschinkel** [comment]
The following as some potentials for ASCII Arts.  What's your favorite?


```


██╗    ██╗██████╗ ██╗     ██╗██████╗     ██████╗  ██████╗ ██╗  ██╗
██║    ██║██╔══██╗██║     ██║██╔══██╗    ██╔══██╗██╔═══██╗╚██╗██╔╝
██║ █╗ ██║██████╔╝██║     ██║██████╔╝    ██████╔╝██║   ██║ ╚███╔╝
██║███╗██║██╔═══╝ ██║     ██║██╔══██╗    ██╔══██╗██║   ██║ ██╔██╗
╚███╔███╔╝██║     ███████╗██║██████╔╝    ██████╔╝╚██████╔╝██╔╝ ██╗
 ╚══╝╚══╝ ╚═╝     ╚══════╝╚═╝╚═════╝     ╚═════╝  ╚═════╝ ╚═╝  ╚═╝

 _ _ _  ___  _    _  _     ___          
| | | || . \| |  <_>| |_  | . > ___ __  
| | | ||  _/| |_ | || . \ | . \/ . \\ \/
|__/_/ |_|  |___||_||___/ |___/\___//\_\
                                        

                                                                                              
                                                                                         
`7MMF'     A     `7MF'`7MM"""Mq.`7MMF'        OO *MM            `7MM"""Yp,                    
  `MA     ,MA     ,V    MM   `MM. MM              MM              MM    Yb                    
   VM:   ,VVM:   ,V     MM   ,M9  MM        `7MM  MM,dMMb.        MM    dP  ,pW"Wq.`7M'   `MF'
    MM.  M' MM.  M'     MMmmdM9   MM          MM  MM    `Mb       MM"""bg. 6W'   `Wb `VA ,V'  
    `MM A'  `MM A'      MM        MM      ,   MM  MM     M8       MM    `Y 8M     M8   XMX    
     :MM;    :MM;       MM        MM     ,M   MM  MM.   ,M9       MM    ,9 YA.   ,A9 ,V' VA.  
      VF      VF      .JMML.    .JMMmmmmMMM .JMML.P^YbmdP'      .JMMmmmd9   `Ybmd9'.AM.   .MA.
                                                                                              
                                                

 __        ______  _     _ _       ____            
 \ \      / /  _ \| |   (_) |__   | __ )  _____  __
  \ \ /\ / /| |_) | |   | | '_ \  |  _ \ / _ \ \/ /
   \ V  V / |  __/| |___| | |_) | | |_) | (_) >  < 
    \_/\_/  |_|   |_____|_|_.__/  |____/ \___/_/\_\
                                                   
      __          __      __   __      
|  | |__) |    | |__)    |__) /  \ \_/ 
|/\| |    |___ | |__)    |__) \__/ / \ 
                                       
 ___       ___   _____    _____        _____   ______       ______      ____     __     __  
(  (       )  ) (  __ \  (_   _)      (_   _) (_   _ \     (_   _ \    / __ \   (_ \   / _) 
 \  \  _  /  /   ) )_) )   | |          | |     ) (_) )      ) (_) )  / /  \ \    \ \_/ /   
  \  \/ \/  /   (  ___/    | |          | |     \   _/       \   _/  ( ()  () )    \   /    
   )   _   (     ) )       | |   __     | |     /  _ \       /  _ \  ( ()  () )    / _ \    
   \  ( )  /    ( (      __| |___) )   _| |__  _) (_) )     _) (_) )  \ \__/ /   _/ / \ \_  
    \_/ \_/     /__\     \________/   /_____( (______/     (______/    \____/   (__/   \__) 
                                                                                            
 __      __  ____    __           __          ____                     
/\ \  __/\ \/\  _`\ /\ \       __/\ \        /\  _`\                   
\ \ \/\ \ \ \ \ \L\ \ \ \     /\_\ \ \____   \ \ \L\ \    ___   __  _  
 \ \ \ \ \ \ \ \ ,__/\ \ \  __\/\ \ \ '__`\   \ \  _ <'  / __`\/\ \/'\ 
  \ \ \_/ \_\ \ \ \/  \ \ \L\ \\ \ \ \ \L\ \   \ \ \L\ \/\ \L\ \/>  </ 
   \ `\___x___/\ \_\   \ \____/ \ \_\ \_,__/    \ \____/\ \____//\_/\_\
    '\/__//__/  \/_/    \/___/   \/_/\/___/      \/___/  \/___/ \//\/_/
                                                                       
                                                                       
oooooo   oooooo     oooo ooooooooo.   ooooo         o8o   .o8            oooooooooo.                        
 `888.    `888.     .8'  `888   `Y88. `888'         `"'  "888            `888'   `Y8b                       
  `888.   .8888.   .8'    888   .d88'  888         oooo   888oooo.        888     888  .ooooo.  oooo    ooo 
   `888  .8'`888. .8'     888ooo88P'   888         `888   d88' `88b       888oooo888' d88' `88b  `88b..8P'  
    `888.8'  `888.8'      888          888          888   888   888       888    `88b 888   888    Y888'    
     `888'    `888'       888          888       o  888   888   888       888    .88P 888   888  .o8"'88b   
      `8'      `8'       o888o        o888ooooood8 o888o  `Y8bod8P'      o888bood8P'  `Y8bod8P' o88'   888o 
                                                                                                            
                                                                                                            
                                                                                                            
 _  _  _ ______  _       _ _        ______             
| || || (_____ \| |     (_) |      (____  \            
| || || |_____) ) |      _| | _     ____)  ) ___ _   _ 
| ||_|| |  ____/| |     | | || \   |  __  ( / _ ( \ / )
| |___| | |     | |_____| | |_) )  | |__)  ) |_| ) X ( 
 \______|_|     |_______)_|____/   |______/ \___(_/ \_)
                                                       
                                                                                                             
I8,        8        ,8I  88888888ba   88           88  88              88888888ba                            
`8b       d8b       d8'  88      "8b  88           ""  88              88      "8b                           
 "8,     ,8"8,     ,8"   88      ,8P  88               88              88      ,8P                           
  Y8     8P Y8     8P    88aaaaaa8P'  88           88  88,dPPYba,      88aaaaaa8P'   ,adPPYba,  8b,     ,d8  
  `8b   d8' `8b   d8'    88""""""'    88           88  88P'    "8a     88""""""8b,  a8"     "8a  `Y8, ,8P'   
   `8a a8'   `8a a8'     88           88           88  88       d8     88      `8b  8b       d8    )888(     
    `8a8'     `8a8'      88           88           88  88b,   ,a8"     88      a8P  "8a,   ,a8"  ,d8" "8b,   
     `8'       `8'       88           88888888888  88  8Y"Ybbd8"'      88888888P"    `"YbbdP"'  8P'     `Y8  
                                                                                                             
                                                                                                             
                                                                                                                            )`·.                                                                                                                                                                                                                                                        
 __          _______  _      _ _       ____            
 \ \        / /  __ \| |    (_) |     |  _ \           
  \ \  /\  / /| |__) | |     _| |__   | |_) | _____  __
   \ \/  \/ / |  ___/| |    | | '_ \  |  _ < / _ \ \/ /
    \  /\  /  | |    | |____| | |_) | | |_) | (_) >  < 
     \/  \/   |_|    |______|_|_.__/  |____/ \___/_/\_\
                                                       
                                                       
 __       __  _______   __        __  __              _______                      
/  |  _  /  |/       \ /  |      /  |/  |            /       \                     
$$ | / \ $$ |$$$$$$$  |$$ |      $$/ $$ |____        $$$$$$$  |  ______   __    __ 
$$ |/$  \$$ |$$ |__$$ |$$ |      /  |$$      \       $$ |__$$ | /      \ /  \  /  |
$$ /$$$  $$ |$$    $$/ $$ |      $$ |$$$$$$$  |      $$    $$< /$$$$$$  |$$  \/$$/ 
$$ $$/$$ $$ |$$$$$$$/  $$ |      $$ |$$ |  $$ |      $$$$$$$  |$$ |  $$ | $$  $$<  
$$$$/  $$$$ |$$ |      $$ |_____ $$ |$$ |__$$ |      $$ |__$$ |$$ \__$$ | /$$$$  \ 
$$$/    $$$ |$$ |      $$       |$$ |$$    $$/       $$    $$/ $$    $$/ /$$/ $$  |
$$/      $$/ $$/       $$$$$$$$/ $$/ $$$$$$$/        $$$$$$$/   $$$$$$/  $$/   $$/ 
                                                                                   
                                                                                   
                                                                                   
 __       __  _______   __        __  __              _______                      
|  \  _  |  \|       \ |  \      |  \|  \            |       \                     
| $$ / \ | $$| $$$$$$$\| $$       \$$| $$____        | $$$$$$$\  ______   __    __ 
| $$/  $\| $$| $$__/ $$| $$      |  \| $$    \       | $$__/ $$ /      \ |  \  /  \
| $$  $$$\ $$| $$    $$| $$      | $$| $$$$$$$\      | $$    $$|  $$$$$$\ \$$\/  $$
| $$ $$\$$\$$| $$$$$$$ | $$      | $$| $$  | $$      | $$$$$$$\| $$  | $$  >$$  $$ 
| $$$$  \$$$$| $$      | $$_____ | $$| $$__/ $$      | $$__/ $$| $$__/ $$ /  $$$$\ 
| $$$    \$$$| $$      | $$     \| $$| $$    $$      | $$    $$ \$$    $$|  $$ \$$\
 \$$      \$$ \$$       \$$$$$$$$ \$$ \$$$$$$$        \$$$$$$$   \$$$$$$  \$$   \$$
                                                                                   
                                                                                   
                                                                                   
$$\      $$\ $$$$$$$\  $$\       $$\ $$\             $$$$$$$\                      
$$ | $\  $$ |$$  __$$\ $$ |      \__|$$ |            $$  __$$\                     
$$ |$$$\ $$ |$$ |  $$ |$$ |      $$\ $$$$$$$\        $$ |  $$ | $$$$$$\  $$\   $$\ 
$$ $$ $$\$$ |$$$$$$$  |$$ |      $$ |$$  __$$\       $$$$$$$\ |$$  __$$\ \$$\ $$  |
$$$$  _$$$$ |$$  ____/ $$ |      $$ |$$ |  $$ |      $$  __$$\ $$ /  $$ | \$$$$  / 
$$$  / \$$$ |$$ |      $$ |      $$ |$$ |  $$ |      $$ |  $$ |$$ |  $$ | $$  $$<  
$$  /   \$$ |$$ |      $$$$$$$$\ $$ |$$$$$$$  |      $$$$$$$  |\$$$$$$  |$$  /\$$\ 
\__/     \__|\__|      \________|\__|\_______/       \_______/  \______/ \__/  \__|
                                                                                   
                                                                                   
                                                                                   
 /$$      /$$ /$$$$$$$  /$$       /$$ /$$             /$$$$$$$                     
| $$  /$ | $$| $$__  $$| $$      |__/| $$            | $$__  $$                    
| $$ /$$$| $$| $$  \ $$| $$       /$$| $$$$$$$       | $$  \ $$  /$$$$$$  /$$   /$$
| $$/$$ $$ $$| $$$$$$$/| $$      | $$| $$__  $$      | $$$$$$$  /$$__  $$|  $$ /$$/
| $$$$_  $$$$| $$____/ | $$      | $$| $$  \ $$      | $$__  $$| $$  \ $$ \  $$$$/ 
| $$$/ \  $$$| $$      | $$      | $$| $$  | $$      | $$  \ $$| $$  | $$  >$$  $$ 
| $$/   \  $$| $$      | $$$$$$$$| $$| $$$$$$$/      | $$$$$$$/|  $$$$$$/ /$$/\  $$
|__/     \__/|__/      |________/|__/|_______/       |_______/  \______/ |__/  \__/
                                                                                   
                                                                                              bbbbbbbb                                                                        
WWWWWWWW                           WWWWWWWWPPPPPPPPPPPPPPPPP   LLLLLLLLLLL               iiii b::::::b                 BBBBBBBBBBBBBBBBB                                      
W::::::W                           W::::::WP::::::::::::::::P  L:::::::::L              i::::ib::::::b                 B::::::::::::::::B                                     
W::::::W                           W::::::WP::::::PPPPPP:::::P L:::::::::L               iiii b::::::b                 B::::::BBBBBB:::::B                                    
W::::::W                           W::::::WPP:::::P     P:::::PLL:::::::LL                     b:::::b                 BB:::::B     B:::::B                                   
 W:::::W           WWWWW           W:::::W   P::::P     P:::::P  L:::::L               iiiiiii b:::::bbbbbbbbb           B::::B     B:::::B   ooooooooooo xxxxxxx      xxxxxxx
  W:::::W         W:::::W         W:::::W    P::::P     P:::::P  L:::::L               i:::::i b::::::::::::::bb         B::::B     B:::::B oo:::::::::::oox:::::x    x:::::x 
   W:::::W       W:::::::W       W:::::W     P::::PPPPPP:::::P   L:::::L                i::::i b::::::::::::::::b        B::::BBBBBB:::::B o:::::::::::::::ox:::::x  x:::::x  
    W:::::W     W:::::::::W     W:::::W      P:::::::::::::PP    L:::::L                i::::i b:::::bbbbb:::::::b       B:::::::::::::BB  o:::::ooooo:::::o x:::::xx:::::x   
     W:::::W   W:::::W:::::W   W:::::W       P::::PPPPPPPPP      L:::::L                i::::i b:::::b    b::::::b       B::::BBBBBB:::::B o::::o     o::::o  x::::::::::x    
      W:::::W W:::::W W:::::W W:::::W        P::::P              L:::::L                i::::i b:::::b     b:::::b       B::::B     B:::::Bo::::o     o::::o   x::::::::x     
       W:::::W:::::W   W:::::W:::::W         P::::P              L:::::L                i::::i b:::::b     b:::::b       B::::B     B:::::Bo::::o     o::::o   x::::::::x     
        W:::::::::W     W:::::::::W          P::::P              L:::::L         LLLLLL i::::i b:::::b     b:::::b       B::::B     B:::::Bo::::o     o::::o  x::::::::::x    
         W:::::::W       W:::::::W         PP::::::PP          LL:::::::LLLLLLLLL:::::Li::::::ib:::::bbbbbb::::::b     BB:::::BBBBBB::::::Bo:::::ooooo:::::o x:::::xx:::::x   
          W:::::W         W:::::W          P::::::::P          L::::::::::::::::::::::Li::::::ib::::::::::::::::b      B:::::::::::::::::B o:::::::::::::::ox:::::x  x:::::x  
           W:::W           W:::W           P::::::::P          L::::::::::::::::::::::Li::::::ib:::::::::::::::b       B::::::::::::::::B   oo:::::::::::oox:::::x    x:::::x 
            WWW             WWW            PPPPPPPPPP          LLLLLLLLLLLLLLLLLLLLLLLLiiiiiiiibbbbbbbbbbbbbbbb        BBBBBBBBBBBBBBBBB      ooooooooooo xxxxxxx      xxxxxxx
                                                                                                                                                                              
                                                                                                                                                                              
                                                                                                                                                                              
                                                                                                                                                                              
                                                                                                                                                                              
                                                                                                                                                                              
                                                                                                                                                                              
q _    _______ _     _ _      ______           
| |  | | ___ \ |   (_) |     | ___ \          
| |  | | |_/ / |    _| |__   | |_/ / _____  __
| |/\| |  __/| |   | | '_ \  | ___ \/ _ \ \/ /
\  /\  / |   | |___| | |_) | | |_/ / (_) >  < 
 \/  \/\_|   \_____/_|_.__/  \____/ \___/_/\_\
                                              

           (   (
 (  (      )\ ))\ )      )     (             
 )\))(   '(()/(()/((  ( /(   ( )\         )  
((_)()\ )  /(_))(_))\ )\())  )((_)  (  ( /(  
_(())\_)()(_))(_))((_|(_)\  ((_)_   )\ )\()) 
\ \((_)/ /| _ \ |  (_) |(_)  | _ ) ((_|(_)\  
 \ \/\/ / |  _/ |__| | '_ \  | _ \/ _ \ \ /  
  \_/\_/  |_| |____|_|_.__/  |___/\___/_\_\  
                                             
 __    __  ___  __ _ _         ___           
/ / /\ \ \/ _ \/ /(_) |__     / __\ _____  __
\ \/  \/ / /_)/ / | | '_ \   /__\/// _ \ \/ /
 \  /\  / ___/ /__| | |_) | / \/  \ (_) >  < 
  \/  \/\/   \____/_|_.__/  \_____/\___/_/\_\
                                             
 _       ______  __    _ __       ____            
| |     / / __ \/ /   (_) /_     / __ )____  _  __
| | /| / / /_/ / /   / / __ \   / __  / __ \| |/_/
| |/ |/ / ____/ /___/ / /_/ /  / /_/ / /_/ />  <  
|__/|__/_/   /_____/_/_.___/  /_____/\____/_/|_|  
                                                  
 __      _____ _    _ _      ___          
 \ \    / / _ \ |  (_) |__  | _ ) _____ __
  \ \/\/ /|  _/ |__| | '_ \ | _ \/ _ \ \ /
   \_/\_/ |_| |____|_|_.__/ |___/\___/_\_\
                                          
  _      _____  __   _ __     ___          
 | | /| / / _ \/ /  (_) /    / _ )___ __ __
 | |/ |/ / ___/ /__/ / _ \  / _  / _ \\ \ /
 |__/|__/_/  /____/_/_.__/ /____/\___/_\_\ 
                                           
 __        ______  _     _ _       ____            
 \ \      / /  _ \| |   (_) |__   | __ )  _____  __
  \ \ /\ / /| |_) | |   | | '_ \  |  _ \ / _ \ \/ /
   \ V  V / |  __/| |___| | |_) | | |_) | (_) >  < 
    \_/\_/  |_|   |_____|_|_.__/  |____/ \___/_/\_\
                                                   
 ____      ____  _______  _____      _   __         ______                    
|_  _|    |_  _||_   __ \|_   _|    (_) [  |       |_   _ \                   
  \ \  /\  / /    | |__) | | |      __   | |.--.     | |_) |   .--.   _   __  
   \ \/  \/ /     |  ___/  | |   _ [  |  | '/'`\ \   |  __'. / .'`\ \[ \ [  ] 
    \  /\  /     _| |_    _| |__/ | | |  |  \__/ |  _| |__) || \__. | > '  <  
     \/  \/     |_____|  |________|[___][__;.__.'  |_______/  '.__.' [__]`\_] 
                                                                              

```

**Comment by clubdeuce** [comment]
````
 __          _______  _      _ _       ____            
 \ \        / /  __ \| |    (_) |     |  _ \           
  \ \  /\  / /| |__) | |     _| |__   | |_) | _____  __
   \ \/  \/ / |  ___/| |    | | '_ \  |  _ < / _ \ \/ /
    \  /\  /  | |    | |____| | |_) | | |_) | (_) >  < 
     \/  \/   |_|    |______|_|_.__/  |____/ \___/_/\_\
````

**Comment by mikeschinkel** [comment]
@clubdeuce Love it!

**Comment by MickMakes** [comment]
It's now in 0.16.0-rc.1, which will be released shortly as 0.16.
